### PR TITLE
Replace `done` callbacks in the font-tests with async/await instead

### PR DESCRIPTION
**Pull request by Snuffleupagus**

### CODE REVIEW

File: test/font/fontutils.js [hunk 1]
```diff
@@ -65,24 +65,26 @@ function encodeFontData(data) {
   return buffer;
 }
 
-function ttx(data, callback) {
-  const xhr = new XMLHttpRequest();
-  xhr.open("POST", "/ttx");
+function ttx(data) {
+  return new Promise((resolve, reject) => {
```
Comment (timvandermeij) [inline test/font/fontutils.js:69]: Can't we just make this function `async` instead so we don't need to wrap it in a `Promise` manually?

**Review by timvandermeij — COMMENTED** [review]
Looks good to me with the comment addressed.

File: test/font/fontutils.js (hunk 1 above)
Comment (Snuffleupagus) [reply test/font/fontutils.js:69]: Unfortunately I cannot see how that'd work, given that `xhr.onreadystatechange` is a *callback* function that we cannot simply `await` :-(

We could *maybe* re-factor this function to use `fetch` instead, assuming this isn't using e.g. a `file:`-URL, but that's probably better left for a follow-up.

Comment (Snuffleupagus) [reply test/font/fontutils.js]: It seems that using `fetch` instead worked just fine, so I've pushed a separate commit that fixes this :-)

**Review by timvandermeij — APPROVED** [review]

**Comment by timvandermeij** [comment]
Looks really good, and using the fetch API is also much cleaner. Thank you for doing this!

**Pull request merged**